Sources/_InternalTestSupport/misc.swift

```diff
@@ -32,12 +32,12 @@ import Testing
 import func XCTest.XCTFail
 import struct XCTest.XCTSkip
 
+import class TSCBasic.Process
```

Comment:
issue: Can we use `AsyncProcess` located in SwiftPM instead of depending on a TSC API.